modules/site-health/admin-site-health.php

```diff
@@ -172,6 +184,43 @@ public function info_options( $debug_info ) {
 		return $debug_info;
 	}
 
+	/**
+	 * Get an array with post_type as key and post IDs as value
+	 *
+	 * @param int $limit  Nb of post max to show per post type.
+	 * @return array
+	 *
+	 * @since   3.0
+	 */
```

Comment:
Could you use the same format as usual `@since` above `@param`. Also provide more precision for the type of array `int[][]`.

Reply:
I'm not sure about how to write the type of the array ...

Reply:
As for @param, I think. The returned array is an array of int: post IDs